test/metabase/test/util.clj

```diff
@@ -671,11 +670,11 @@
   "Additional conditions applied to the query to find the max ID for a model prior to a test run. This can be used to
   exclude rows which intentionally use non-sequential IDs, like the internal user."
   {:arglists '([model])}
-  mi/model)
+  identity)
```

Comment:
IIRC `mi/model` exists so that `'User` or `User` will be resolved to `:model/User`. so probably safer to keep it.

Reply:
Uhh, I'm not even sure what happened here. 😳 Fixed in b3d0e32fcf

Reply:
Aha, mi/model resolves `:model/User` to nil. Actually, I can't seem to make it to return anything sensible:

```
metabase.test.util> (mi/model 'User)
nil
metabase.test.util> (mi/model metabase.models.user/User)
nil
metabase.test.util> (mi/model :model/User)
nil
```

Reply:
huh, what!

Reply:
anw, I think you can use `t2/resolve-model`

Reply:
Oh, right. I wonder how it worked before?..